tools/map/build_terrain_grid.py

```
@@ -0,0 +1,803 @@
+    영역은 그 縣들이 가진 셀의 합집합으로 만든다.
+
+    맞춤은 좌표가 먼저다. 郡國志의 縣 이름과 CHGIS 이름은 자주 어긋난다(雒陽/洛陽 같은
+    한대-후대 표기 차이). 이름은 같은 거리의 후보를 가를 때만 쓴다.
+    """
+    import math
+
+    # 이체자. CHGIS 표기와 郡國志 표기가 갈리면 같은 郡이 둘로 쪼개져 서로의 땅에
+    # 위요지를 만든다(郁林/鬱林, 河間/河閒). 접기 전에 한 글자로 맞춘다.
+    var = str.maketrans('郁閒雒鴈沇涼峕竝恒鬴', '鬱間洛雁兗凉時并弘釜')
+
+    def norm(n):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `norm`

(ANN202)